report/report-generators/src/main/kotlin/com/testerum/report_generators/reports/console_debug/ConsoleDebugExecutionListener.kt

```diff
@@ -15,14 +17,17 @@ import com.testerum.runner.events.model.SuiteStartEvent
 import com.testerum.runner.events.model.TestEndEvent
 import com.testerum.runner.events.model.TestStartEvent
 import com.testerum.runner.events.model.TextLogEvent
-import com.testerum.runner_cmdline.events.execution_listeners.utils.console_output_capture.ConsoleOutputCapturer
 import javax.annotation.concurrent.NotThreadSafe
 
 @NotThreadSafe
 class ConsoleDebugExecutionListener : BaseExecutionListener() {
 
     private var indentLevel = 0
 
+    override fun onConfigurationEvent(event: ConfigurationEvent) {
+        log("CONFIGURATION: ${event.toString()}\n")
```

Comment:
No need for ``toString``. We can have this instead:
```suggestion
        log("CONFIGURATION: $event\n")
```

Reply:
done